File: leetcode/array/485. Max Consecutive Ones/Solution.java
public class Solution {
    public static int findMaxConsecutiveOnes(int[] nums) {
        int count = 0, res = 0;
        for (int i : nums) {
            if (i == 0) {
                count = 0;
                continue;
            }
            count++;
            res = res > count ? res : count;
        }
        return res;
    }

    public static void main(String[] args) {
        int[] input = { 1, 0, 1, 1, 0, 1 };
        System.out.println(findMaxConsecutiveOnes(input));
    }
}

File: leetcode/array/485. Max Consecutive Ones/Solution.js
const findMaxConsecutiveOnes = nums => {
    let count = 0,
        res = 0;
    for (let i of nums) {
        if (i === 0)
            count = 0;
        else {
            count++;
            res = res > count ? res : count;
        }
    }
    return res;
};
console.log(findMaxConsecutiveOnes([1, 1, 0, 1, 1, 1]));